cloudamqp/resource_cloudamqp_vpc_peering.go

```diff
@@ -59,7 +72,7 @@ func resourceVpcPeeringAccept(d *schema.ResourceData, meta interface{}) error {
 		d.SetId(d.Get("peering_id").(string))
 	}
 
-	return nil
+	return resourceVpcPeeringRead(d, meta)
```

Comment:
Why this change?

Reply:
Previous missed to populate status schema attribute during accept VPC peering.